apps/web/modules/ui/components/input-combo-box/index.tsx

```
@@ -154,186 +150,225 @@ export const InputCombobox = ({
             id={id}
             role="combobox"
             aria-controls="options"
             aria-expanded={open}
-            className={cn(
-              "flex h-10 w-full shrink-0 cursor-pointer items-center justify-end rounded-md bg-white pr-2",
-              { "w-10 justify-center pr-0": withInput && inputType !== "dropdown" }
-            )}>
+            className={cn("flex h-10 w-full cursor-pointer items-center justify-end bg-white pr-2", {
+              "w-10 justify-center pr-0": withInput && inputType !== "dropdown",
+              "pointer-events-none": isClearing,
+            })}>
```

Biome (1.9.4)

[error] The HTML element with the interactive role "combobox" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "combobox".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)